rmf_site_editor/src/site/load.rs

```diff
@@ -367,50 +367,37 @@ fn generate_site_entities(
             .convert(&id_to_entity)
             .for_site(site_id)?;
 
-        if let Some(model_description) = model_instance.description.0 {
-            let source = model_description_to_source.get(&model_description);
-            if let Some(mut loaded_instance) =
-                model_loader.spawn_model_instance(site_id, model_instance.clone(), source)
-            {
-                let model_instance_entity = loaded_instance
-                    .insert((Category::Model, SiteID(*model_instance_id)))
-                    .id();
-                id_to_entity.insert(*model_instance_id, model_instance_entity);
-                consider_id(*model_instance_id);
-
-                if let Some(instances) = model_description_dependents.get_mut(&model_description) {
-                    instances.insert(model_instance_entity);
-                } else {
-                    error!(
-                        "Model description missing for instance {}. This should \
+        let model_instance_entity = model_loader
+            .spawn_model_instance(site_id, model_instance.clone())
+            .insert((Category::Model, SiteID(*model_instance_id)))
+            .id();
+        id_to_entity.insert(*model_instance_id, model_instance_entity);
+        consider_id(*model_instance_id);
+
+        if let Some(instances) = model_instance
+            .description
+            .0
+            .map(|e| model_description_dependents.get_mut(&e))
+            .flatten()
+        {
+            instances.insert(model_instance_entity);
+        } else {
+            error!(
+                "Model description missing for instance {}. This should \
                         not happen, please report this bug to the maintainers of \
```

Comment:
Minor nitpick: Let's get this text aligned with the rest of the code.

Reply:
[6067fef](https://github.com/open-rmf/rmf_site/pull/242/commits/6067feffabf66bd01dcb2e8c3537e8937bb041fb)